resources/load-staticly-from-npm.js

```diff
@@ -15,23 +15,20 @@ const fs = require('fs');
  *
  *  "<file content>"
  */
-module.exports = function inlineInvariant(context) {
-  return {
-    visitor: {
-      CallExpression(path) {
-        const { node } = path;
-
-        if (
-          node.callee.type === 'Identifier' &&
-          node.callee.name === 'loadFileStaticlyFromNPM'
-        ) {
-          const npmPath = node.arguments[0].value;
-          const filePath = require.resolve(npmPath);
-          const content = fs.readFileSync(filePath, 'utf-8');
-
-          path.replaceWith(context.types.stringLiteral(content));
-        }
-      },
-    },
+module.exports = function transformLoadStaticlyFromNPM(context) {
+  const visit = (node) => {
+    if (ts.isCallExpression(node)) {
+      if (
+        ts.isIdentifier(node.expression) &&
+        node.expression.text === 'loadFileStaticlyFromNPM'
+      ) {
+        const npmPath = node.arguments[0].text;
+        const filePath = require.resolve(npmPath);
+        const content = fs.readFileSync(filePath, 'utf-8');
+        return ts.createStringLiteral(content);
+      }
+    }
+    return ts.visitEachChild(node, visit, context);
   };
+  return visit;
```

Comment:
Why you can't just return arrow function?

Reply:
It's recursive. I'll change it to a regular function.

Reply:
Ok, I see, thanks for the explanation 👍